test/kpod.bats

```diff
@@ -47,3 +48,61 @@ KPOD_OPTIONS="--root $ROOT --runroot $RUNROOT --storage-driver vfs"
 	echo "$output"
 	[ "$status" -ne 0 ]
 }
+
+@test "kpod history default" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} history $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+}
+
+@test "kpod history with format" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} history --format "{{.ID}} {{.Created}}" $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+
+}
+
+@test "kpod history human flag" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} history --human=false $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+
+}
+
+@test "kpod history quiet flag" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} history -q $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+
+}
+
+@test "kpod history no-trunc flag" {
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} pull $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+	run ${KPOD_BINARY} ${KPOD_OPTIONS} history --no-trunc $IMAGE
+	echo "$output"
+	[ "$status" -eq 0 ]
+
+}
+
+@test "kpod history json flag" {
```

Comment:
Perhaps check if the json is well formatted.
/usr/bin/json_verify
or 
/usr/bin/json_pp